src/DetailsView/components/issue-filing-dialog.tsx

```diff
@@ -8,10 +8,11 @@ import { BugFilingService } from '../../bug-filing/types/bug-filing-service';
 import { EnvironmentInfoProvider } from '../../common/environment-info-provider';
 import { NamedSFC } from '../../common/react/named-sfc';
 import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { IssueFilinigNeedsSettingsContentProps } from '../../common/types/issue-filing-needs-setting-content';
 import { BugServiceProperties } from '../../common/types/store-data/user-configuration-store';
 import { ActionAndCancelButtonsComponent } from './action-and-cancel-buttons-component';
 
-export interface IssueFilingDialogProps {
+export interface IssueFilingDialogProps extends IssueFilinigNeedsSettingsContentProps {
```

Comment:
I wonder if this is backwards - should it be that `IssueFilinigNeedsSettingsContentProps` extends `IssueFilingDialogProps`? (see comment at line 57 below)

Reply:
right, Make sense!